firebase-config/src/test/java/com/google/firebase/remoteconfig/internal/ConfigFetchHttpClientTest.java

```diff
@@ -208,6 +208,20 @@ public void fetch_setsAllElementsOfRequestBody_sendsRequestBodyToServer() throws
         .isEqualTo(new JSONObject(userProperties).toString());
   }
 
+  @Test
+  public void fetch_requestEncodesLanguageSubtags() throws Exception {
```

Comment:
Can we add a similar test for a device that's below API Level 21? Thanks!

Reply:
Good call!